### PR TITLE
Docs/update local run readme

### LINKED ISSUE #78: Can't start docker-compose: missing .env file
**Describe the bug**

`docker-compose up` fails with the following error:

```
$ docker-compose up
ERROR: Couldn't find env file: /Users/marcelloromani/dev/xcov19/covidX/.env
```

### LINKED ISSUE #79: docker-compose freezing after building successfully
**Describe the bug**
After building successfully, `docker-compose` freezes on the following section:
```
(18:29:37) INFO: Analyzed target //:manage (0 packages loaded, 0 targets configured).
(18:29:37) INFO: Found 1 target...
Target //:manage up-to-date:
  bazel-bin/manage
  bazel-bin/manage.zip
(18:29:37) INFO: Elapsed time: 1.356s, Critical Path: 0.01s
(18:29:37) INFO: 1 process: 1 internal.
(18:29:37) INFO: Build completed successfully, 1 total action
(18:29:37) INFO: Build completed successfully, 1 total action
django-api  | 2021-01-25 18:29:42,488 - Logger - INFO - Starting covidX app
django-api  | 2021-01-25 18:29:42,489 - Logger - INFO - BASE_DIR= /tmp/Bazel.runfiles_zcnxqi2m/runfiles/covidx
django-api  | System check identified some issues:
django-api  |
django-api  | WARNINGS:
django-api  | ?: (urls.W001) Your URL pattern '^apihealth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^login-rest-router/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^rest-auth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | 2021-01-25 18:29:46,783 - Logger - INFO - Starting covidX app
django-api  | 2021-01-25 18:29:46,783 - Logger - INFO - BASE_DIR= /tmp/Bazel.runfiles_zcnxqi2m/runfiles/covidx
django-api  | System check identified some issues:
django-api  |
django-api  | WARNINGS:
django-api  | ?: (urls.W001) Your URL pattern '^apihealth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^login-rest-router/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  | ?: (urls.W001) Your URL pattern '^rest-auth/?$' uses include with a route ending with a '$'. Remove the dollar from the route to avoid problems including URLs.
django-api  |
django-api  | System check identified 3 issues (0 silenced).

```

**To Reproduce**
1. Clone the repository
2. Add a `.env` file in the cloned repository.
3. Run `docker-compose up --build` 
4. Wait for building to complete
5. See error

**Expected behavior**
Build completed successfully and command terminates

**Screenshots**
The build process freezes upon reaching this section:
image

**Desktop:**
 - OS: Windows 10

### PR DESCRIPTION
**Pull request by marcelloromani**
<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/82"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/82)

<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/82"></a>

### CODE REVIEW

**Comment by marcelloromani** [comment]
Should fix #78

File: Readme.md [hunk 1]
```diff
@@ -15,24 +15,52 @@ TL;DR Dive into coding right away by clicking here:
...
+* Fill in the details based on the Auth0 and Algolia sign-ups
+
+See https://github.com/Xcov19/covidX/issues/78
+
+* Generate a private / public key pair
+
 ```shell
-docker run --rm -it -e SECRET_KEY=<YOUR_DJANGO_SECRET_KEY> -e DEBUG_ENV=1 -e CPPFLAGS="$(pg_config --cppflags)" -e LDFLAGS="$(pg_config --ldflags)" -e SECRET_ID="SECRET_KEY" -e BUCKET_NAME="gae-bizlead" -e DJANGO_SETTINGS_MODULE="covidX.settings" -e WSGI_APPLICATION="covidX.wsgi.application" --security-opt apparmor=unconfined codecakes/covidx_manage:bazel runserver_plus
+openssl req -x509 -newkey rsa:4096 -keyout privateKey.key -out certificate -days 365 -nodes
 ```
```
Comment (milan-tom) [inline Readme.md:46]: Could we do this in [docker itself](https://codefresh.io/docker-tutorial/using-docker-generate-ssl-certificates/)?

File: Readme.md [hunk 2]
```diff
@@ -15,24 +15,52 @@ TL;DR Dive into coding right away by clicking here:
...
+docker-compose up
+```
+
+The app should be built automatically on first run.
+
+Note: to reflect changes made to `.env` after the application has been brought `up`, you need to force a rebuild:
+
+```shell
+docker-compose --build
+```
```
Comment (milan-tom) [inline Readme.md:60]: @codecakes Is this [previous section](https://github.com/marcelloromani/covidX/blame/docs/update_local_run_readme/Readme.md#L109-L124) needed now that we can simply do `docker-compose up`?

File: Readme.md (hunk 1 above)
Comment (marcelloromani) [reply Readme.md:46]: FWIW I like the idea, I think it could be turned into a new ticket.

Comment (milan-tom) [reply Readme.md:46]: Yes, maybe it's best to merge this and work on it in a separate pull request. That way the current readme will be up to date for others trying to build.

File: Readme.md (hunk 2 above)
Comment (codecakes) [reply Readme.md:60]: docker build has nothing to do with previous section. ive already told to rename previous section for developers. "If you don't know bazel, dont bothe with this section" @marcelloromani @milan-tom

Comment (marcelloromani) [reply Readme.md:61]: Added that line, hopefully in the right place.

**Review by codecakes — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (milan-tom) [reply Readme.md:47]: Implemented in #100

**Review by milan-tom — APPROVED** [review]

**Review by codecakes — APPROVED** [review]

**Comment by codecakes** [comment]
closes #78

**Pull request merged**